guides/logging.yaml

```diff
@@ -1,18 +1,16 @@
-TOPIC
-    logging
-
-DESCRIPTION
+topic: logging
+guide: |
     Bolt prints messages both to the console and to log files. Messages can
     either come from Bolt's 'outputter', which logs user-facing messages like
     progress and results, or from the 'logger', which logs warnings, errors, and
     log-structured output to log files. Both of these message streams are
     configurable.
 
     By default, Bolt logs to the console at 'warn' level and writes a log file to
-    '<project>/bolt-debug.log' at 'debug' level. Unless you are running a plan,
+    '<project|/bolt-debug.log' at 'debug' level. Unless you are running a plan,
```

Comment:
Same here; why the change to a pipe?

Reply:
I think I used a sed command to find + replace that must have modified these....